src/client/src/App/Credit/SellSide/SellSideTradeTicket/SellSideTradeTickerFooter.tsx

```diff
@@ -145,6 +159,7 @@ export const SellSideTradeTicketFooter = ({
       {state === RfqState.Open && (
         <>
           <PassButton
+            tabIndex={2}
```

Comment:
per chat .. reconsider when Pass comes online .. for now, I would remove these `tabIndex` overrides and go with default flow - and disable `PassButton` please (should have been done last year!) .. especially as we have Enter-to-submit-quote in Price field